<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>KevaDialog</class>
  <widget class="QWidget" name="KevaDialog">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>776</width>
        <height>396</height>
      </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
      <item>
        <widget class="QFrame" name="frame2">
          <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
              <layout class="QGridLayout" name="gridLayout">
                <item row="4" column="2">
                  <layout class="QHBoxLayout" name="nsHLayout">
                    <item>
                      <widget class="QLineEdit" name="nameSpace">
                        <property name="toolTip">
                          <string>The namespace ID with a prefix "N".</string>
                        </property>
                        <property name="maximumWidth">
                          <number>600</number>
                        </property>
                        <property name="sizePolicy">
                          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                          </sizepolicy>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="bookmarkNamespace">
                        <property name="toolTip">
                          <string>Bookmark this namespace</string>
                        </property>
                        <property name="icon">
                          <iconset resource="../bitcoin.qrc">
                            <normaloff>:/icons/star_empty</normaloff>:/icons/star_empty
                          </iconset>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <spacer name="horizontalBookmarkSpacer">
                        <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizePolicy">
                          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                          </sizepolicy>
                        </property>
                      </spacer>
                    </item>
                  </layout>
                </item>
                <item row="2" column="2">
                  <widget class="QLabel" name="label_5">
                    <property name="text">
                      <string>Use this form to perform Keva database operations.</string>
                    </property>
                  </widget>
                </item>
                <item row="4" column="0">
                  <widget class="QLabel" name="label_2">
                    <property name="toolTip">
                      <string>The namespace ID with a prefix "N".</string>
                    </property>
                    <property name="text">
                      <string>Namespace:</string>
                    </property>
                    <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="buddy">
                      <cstring>namespace</cstring>
                    </property>
                  </widget>
                </item>
                <item row="7" column="2">
                  <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                      <widget class="QPushButton" name="showContent">
                        <property name="sizePolicy">
                          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                          </sizepolicy>
                        </property>
                        <property name="toolTip">
                          <string>Show content of the namespace.</string>
                        </property>
                        <property name="text">
                          <string>Show</string>
                        </property>
                        <property name="icon">
                          <iconset resource="../bitcoin.qrc">
                            <normaloff>:/icons/eye</normaloff>:/icons/eye</iconset>
                        </property>
                        <property name="autoDefault">
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="createNamespace">
                        <property name="sizePolicy">
                          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                          </sizepolicy>
                        </property>
                        <property name="toolTip">
                          <string>Create a new namespace</string>
                        </property>
                        <property name="text">
                          <string>&amp;Create namespace</string>
                        </property>
                        <property name="icon">
                          <iconset resource="../bitcoin.qrc">
                            <normaloff>:/icons/add</normaloff>:/icons/add</iconset>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="listNamespaces">
                        <property name="sizePolicy">
                          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                          </sizepolicy>
                        </property>
                        <property name="toolTip">
                          <string>List my namespaces</string>
                        </property>
                        <property name="text">
                          <string>&amp;My Namespaces</string>
                        </property>
                        <property name="icon">
                          <iconset resource="../bitcoin.qrc">
                            <normaloff>:/icons/editpaste</normaloff>:/icons/editpaste</iconset>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="bookmarksButton">
                        <property name="sizePolicy">
                          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                          </sizepolicy>
                        </property>
                        <property name="toolTip">
                          <string>Show bookmarks</string>
                        </property>
                        <property name="text">
                          <string>&amp;Bookmarks</string>
                        </property>
                        <property name="icon">
                          <iconset resource="../bitcoin.qrc">
                            <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <spacer name="horizontalSpacer">
                        <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                          <size>
                            <width>40</width>
                            <height>20</height>
                          </size>
                        </property>
                      </spacer>
                    </item>
                  </layout>
                </item>
                <item row="7" column="0">
                  <widget class="QLabel" name="label_7">
                    <property name="text">
                      <string/>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <spacer name="verticalSpacer_2">
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
            <size>
              <width>20</width>
              <height>10</height>
            </size>
          </property>
        </spacer>
      </item>
      <item>
        <widget class="QFrame" name="frame">
          <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
              <widget class="QLabel" name="label_6">
                <property name="font">
                  <font>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>
                <property name="text">
                  <string>Content of namespace</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QTableView" name="kevaView">
                <property name="contextMenuPolicy">
                  <enum>Qt::CustomContextMenu</enum>
                </property>
                <property name="tabKeyNavigation">
                  <bool>false</bool>
                </property>
                <property name="sortingEnabled">
                  <bool>true</bool>
                </property>
              </widget>
            </item>
            <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                  <widget class="QPushButton" name="createButton">
                    <property name="enabled">
                      <bool>false</bool>
                    </property>
                    <property name="toolTip">
                      <string>Create new record</string>
                    </property>
                    <property name="text">
                      <string>Create</string>
                    </property>
                    <property name="icon">
                      <iconset resource="../bitcoin.qrc">
                        <normaloff>:/icons/add</normaloff>:/icons/add</iconset>
                    </property>
                    <property name="autoDefault">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QPushButton" name="readButton">
                    <property name="enabled">
                      <bool>false</bool>
                    </property>
                    <property name="toolTip">
                      <string>Read selected record</string>
                    </property>
                    <property name="text">
                      <string>Read</string>
                    </property>
                    <property name="icon">
                      <iconset resource="../bitcoin.qrc">
                        <normaloff>:/icons/eye</normaloff>:/icons/eye</iconset>
                    </property>
                    <property name="autoDefault">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QPushButton" name="updateButton">
                    <property name="enabled">
                      <bool>false</bool>
                    </property>
                    <property name="toolTip">
                      <string>Update selected record</string>
                    </property>
                    <property name="text">
                      <string>Update</string>
                    </property>
                    <property name="icon">
                      <iconset resource="../bitcoin.qrc">
                        <normaloff>:/icons/edit</normaloff>:/icons/edit</iconset>
                    </property>
                    <property name="autoDefault">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QPushButton" name="deleteButton">
                    <property name="enabled">
                      <bool>false</bool>
                    </property>
                    <property name="toolTip">
                      <string>Delete the selected entries from the list</string>
                    </property>
                    <property name="text">
                      <string>Delete</string>
                    </property>
                    <property name="icon">
                      <iconset resource="../bitcoin.qrc">
                        <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
                    </property>
                    <property name="autoDefault">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                      <size>
                        <width>40</width>
                        <height>20</height>
                      </size>
                    </property>
                  </spacer>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>BitcoinAmountField</class>
      <extends>QLineEdit</extends>
      <header>qt/bitcoinamountfield.h</header>
      <container>1</container>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>reqMessage</tabstop>
    <tabstop>receiveButton</tabstop>
    <tabstop>clearButton</tabstop>
    <tabstop>recentRequestsView</tabstop>
    <tabstop>readButton</tabstop>
    <tabstop>deleteButton</tabstop>
  </tabstops>
  <resources>
    <include location="../bitcoin.qrc"/>
  </resources>
  <connections/>
</ui>
